src/scss/_mixins.scss

```diff
@@ -102,7 +108,7 @@
 					$tint: _oColorsHSB($hue, $saturation, $value);
 				}
 
-				@include oColorsSetColor($name, $tint);
+				@include oColorsSetColor($name, ($tint, $deprecated));
```

Comment:
Would be great to update the `@example` and at `@param` documentation of `oColorsSetColor` to indicate it should accept a list.